hooks/apply-commit-message-template/apply-commit-message-template

```diff
@@ -21,4 +22,29 @@ if [ "$PRE_COMMIT_COMMIT_MSG_SOURCE" != "message" ] && [ "$PRE_COMMIT_COMMIT_MSG
 
     cat "$full_path" > .git/COMMIT_EDITMSG
 
-fi
+
+}
+
+
+case "$PRE_COMMIT_COMMIT_MSG_SOURCE" in
+
+    "message")
+        # Do nothing, this is when you commit with -m
+    ;;
+    "commit")
+        # Do nothing, not sure when this is called??? -> todo: find this out
```

Comment:
should we publish this comment? maybe create an issue in our Jira?

Reply:
Or I should just find this out now. Hopefully won't take too long. I put it there at some point but can't remember why. The pre-commit documentation for this environment variable is not so good

Reply:
Okay I actually found some documentation: https://git-scm.com/docs/githooks#_prepare_commit_msg -> it is the same as the second argument here, so this is what I need to implement here, and leave a comment pointing to this source too

- [ ] TODO